README.md

```diff
@@ -74,6 +76,12 @@ Also, you can package `build/unpacked` to whatever you want, like **.deb** with:
 
 ```sh
 dpkg-deb --root-owner-group --build unpacked
+
+> [!NOTE]
+> For Debian systems:
+> 1. Copy build/DEBIAN/ to build/unpacked
+> 2. In the control file, replace the liblightdm-gobject-1-dev dependency with liblightdm-gobject-dev.
+> the liblightdm-gobject-1-dev package only exists in Ubuntu, the Debian equivalent is liblightdm-gobject-dev.
 ```
```

Comment:
Better to keep the note outside of the code block.

Reply:
Thanks for the patience... Fixed